### PR TITLE
New element/mc 2399 minimap

### PR DESCRIPTION
**Pull request by Lelith**
Who: @lelith
What / Why: new reusable MiniMap component based on the google maps api.
https://mcmakler.atlassian.net/browse/MC-2399

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +41,7 @@
     "babel-preset-react": "6.11.1",
     "babel-preset-stage-2": "6.13.0",
     "css-loader": "0.23.1",
-    "eslint": "^4.6.1",
+    "eslint": "^3.15.0",
```
Comment (TejasQ) [inline package.json]: A new map feature needs an older eslint? What?

File: src/components/App/MiniMapPage/MiniMapPage.component.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import Page from '../Page/Page.component';
+import PreviewBox from '../PreviewBox/PreviewBox.component';
+import CodeSnippet from '../CodeSnippet/CodeSnippet.component';
+import MiniMap from '../MiniMapComponent/MiniMap.component';
+
+import { html1, html2 } from './snippets/minimap.snippet';
```
Comment (TejasQ) [inline src/components/App/MiniMapPage/MiniMapPage.component.jsx]: Are these good names?

**Comment by baruchvlz** [comment]
@TejasQ Look at you being all caring about your baby!

@Lelith Please verify Tejas's review.

**Comment by TejasQ** [comment]
LGTM

**Pull request merged**